distribution/src/main/resources/bin/elasticsearch

```diff
@@ -53,6 +54,12 @@ EOF
     exit 1
 fi
 
+parse_jvm_options() {
+  if [ -f "$1" ]; then
+    echo "$(grep "^-" "$1" | tr '\n' ' ')"
```

Comment:
@dakrone That would cause `grep` to interpret the pattern as a fixed string, not as a regular expression but I'm using `^` as the beginning-of-line metacharacter and not as the literal `^`.